protobuf/src/stream.rs

```diff
@@ -737,23 +737,37 @@ impl<'a> CodedOutputStream<'a> {
     }
 
     pub fn write_raw_little_endian32(&mut self, value: u32) -> ProtobufResult<()> {
-        let bytes = unsafe { mem::transmute::<_, [u8; 4]>(value.to_le()) };
+        let bytes: [u8; 4] = [
+            ((value >>  0) & 0xff) as u8,
+            ((value >>  8) & 0xff) as u8,
+            ((value >> 16) & 0xff) as u8,
+            ((value >> 24) & 0xff) as u8,
+        ];
+
         self.write_raw_bytes(&bytes)
     }
 
     pub fn write_raw_little_endian64(&mut self, value: u64) -> ProtobufResult<()> {
-        let bytes = unsafe { mem::transmute::<_, [u8; 8]>(value.to_le()) };
```

Comment:
This can be replaced with `value.to_le_bytes()` in modern Rust.

Reply:
@stepancheg Good suggestion, I didn't know that method existed!

Reply:
I cherry-picked this commit and included `to_le_bytes()` change, committed as b118f2435a045251bf2cb1afd56abaf20985f512.